CrazyCanvas/Source/GUI/LobbyGUI.cpp

```diff
@@ -51,12 +52,19 @@ void LobbyGUI::InitGUI(LambdaEngine::String name)
 {
 	strcpy(m_GameSettings.ServerName, (name + "'s server").c_str());
 
+	LambdaEngine::TArray<LambdaEngine::String> gameModeNames;
+	LambdaEngine::TArray<EGameMode> gameModes;
+	GameModesQuery(gameModes);
+	for (EGameMode gameMode : gameModes)
+		gameModeNames.PushBack(GameModeToString(gameMode));
+
```

Comment:
gameModeNames could be initilized with the size from gameModes to avoid more allocation than needed.